website/blog/2025-02-07-best-alternative-to-mongodb-data-api-and-http-endpoints.md

```diff
@@ -0,0 +1,199 @@
+Open source solutions like FerretDB offer control, community support, and long-term stability, so you're never left stranded.
+
+The good thing is: with the release of FerretDB v2, you can successfully replace MongoDB Data API and perform database operations on FerretDB using a direct HTTP-based method to access and interact with their databases.
+
+In this post, we'll walk you through how to use FerretDB's Data API with Postman, demonstrating how you can query, insert, update, and delete documents – all without needing a MongoDB server.
+
+## Why Data API matters
+
+Some programming environments just don't have a direct way to connect to a database – no native MongoDB driver, no built-in support, nothing.
+Without a driver, you'd normally have to wrap your logic inside another application or depend on a separate service just to send or retrieve data.
+That's extra work and dependencies.
+
+A Data API changes that – suddenly, your database is accessible through simple HTTP requests.
+A basic `curl` command can fetch or modify data.
+So instead of needing a full-fledged backend just to store something in a database, a workflow can send REST requests to a Data API and log information.
+
+For developers already familiar with REST, Data API is essential.
+There's no new learning curve – just familiar HTTP methods (`GET`, `POST`, `PATCH`, `DELETE`).
+With FerretDB's Data API stepping in as an alternative to MongoDB's deprecated service, we are ensuring developers can interact with their data without issues, no matter the stack they're working with.
+
+## FerretDB Data API
+
+FerretDB is an open-source MongoDB-compatible database that runs on PostgreSQL.
+With MongoDB's Data API deprecated, we saw the need for a new solution that offers similar functionality while remaining open and accessible.
+
+FerretDB's Data API provides:
+
+- Seamless HTTP-based database access – No need for drivers; interact via simple REST requests.
+- MongoDB-compatible queries – Use familiar JSON-based queries and commands.
+- PostgreSQL-backed storage – Leverage the reliability of PostgreSQL with a MongoDB-like API.
+
+To access Data API on FerretDB, set the environment variable or flag (`FERRETDB_LISTEN_DATA_API_ADDR`/`--listen-data-api-addr`) to your desired port when starting FerretDB.
+
+## Setting up FerretDB Data API
+
+### Get your FerretDB instance running
+
+You can set it up via Docker compose.
+Here's an example:
+
+```yaml
+services:
+  postgres:
+    image: ghcr.io/ferretdb/postgres-documentdb:16.6-0.100-0
+    restart: on-failure
+    environment:
+      - POSTGRES_USER=username
+      - POSTGRES_PASSWORD=password
+      - POSTGRES_DB=postgres
+    volumes:
+      - ./data:/var/lib/postgresql/data
+
+  ferretdb:
+    image: ghcr.io/ferretdb/ferretdb:2.0.0-rc.1
```

Comment:
What's our blog guideline for using specific release version such as `2.0.0-rc.1`, verses just any version 2?

Reply:
Oops, it should be just the version.